modules/subnets/main.tf

```diff
@@ -33,9 +33,9 @@ resource "google_compute_subnetwork" "subnetwork" {
   private_ip_google_access = lookup(each.value, "subnet_private_access", "false")
   dynamic "log_config" {
     for_each = lookup(each.value, "subnet_flow_logs", false) ? [{
-      aggregation_interval = lookup(each.value, "subnet_flow_logs_interval", null)
-      flow_sampling        = lookup(each.value, "subnet_flow_logs_sampling", null)
-      metadata             = lookup(each.value, "subnet_flow_logs_metadata", null)
+      aggregation_interval = lookup(each.value, "subnet_flow_logs_interval", "INTERVAL_5_SEC")
```

Comment:
What's the reason for making this breaking change? I don't see a strong reason to enforce our own defaults.

Reply:
In 3.x.x within log_config block at least one of aggregation_interval, flow_sampling, or metadata should be set. Hence if subnet_flow_logs is true, I believe the current dynamic block was setting empty blocks which threw at least one aggregation_interval, flow_sampling, or metadata should be set error. These defaults I propose are the same defaults as the provider docs.
[3.0.0 Subnetwork Upgrade Guide](https://www.terraform.io/docs/providers/google/guides/version_3_upgrade.html#resource-google_compute_subnetwork)
[Log Config Defaults](https://www.terraform.io/docs/providers/google/r/compute_subnetwork.html#log_config)

Reply:
Yes, that should work.

Reply:
sg, so I am leaving these defaults as is.